### PR TITLE
more design fixes

### PR DESCRIPTION
**Pull request by SaraVieira**
yay? closer to the edge OF GLORY

### CODE REVIEW

File: packages/app/src/app/pages/Sandbox/Editor/Workspace/screens/Explorer/Files/DirectoryEntry/Entry/FileInput.tsx [hunk 1]
```diff
@@ -31,13 +31,13 @@ export const FileInput: FunctionComponent<Props> = ({
...
   id,
 }) => {
-  const [currentValue, setCurretValue] = useState(title);
+  const [currentValue, setCurrentValue] = useState(title);
 
   const handleChange = (e: ChangeEvent<HTMLInputElement>) => {
     if (e.target) {
       const { value } = e.target;
       onChange(value);
-      setCurretValue(value);
```
Comment (siddharthkp) [inline packages/app/src/app/pages/Sandbox/Editor/Workspace/screens/Explorer/Files/DirectoryEntry/Entry/FileInput.tsx]: me, an intellectual

File: packages/app/src/app/pages/Sandbox/Editor/Workspace/screens/ConfigurationFiles/index.tsx [hunk 2]
```diff
@@ -109,7 +109,11 @@ export const ConfigurationFilesComponent = ({ theme }) => {
                       {config.description}
                     </Text>
                     <Button
-                      style={{ width: 'auto' }}
+                      style={{
+                        width: 'auto',
+                        height: 'auto',
+                        padding: '2px 0',
```
Comment (siddharthkp) [inline packages/app/src/app/pages/Sandbox/Editor/Workspace/screens/ConfigurationFiles/index.tsx]: aye, not a fan of modifying padding of elements from outside :(

waiting for staging build to check out why

Comment (SaraVieira) [reply packages/app/src/app/pages/Sandbox/Editor/Workspace/screens/ConfigurationFiles/index.tsx]: Screenshot 2020-01-31 at 14 26 47

This, want me to make a button size="small"?

**Comment by lbogdan** [comment]
Build for latest commit https://github.com/codesandbox/codesandbox-client/pull/3437/commits/65c2fbcfdec20209c2abbe6b9453f4d0058e54a2 is at https://pr3437.build.csb.dev/s/new.

File: packages/app/src/app/pages/Sandbox/Editor/Workspace/screens/ConfigurationFiles/index.tsx (hunk 2 above)
Comment (SaraVieira) [reply packages/app/src/app/pages/Sandbox/Editor/Workspace/screens/ConfigurationFiles/index.tsx]: Changed to height:4

**Pull request merged**